### PR TITLE
Feature/pagation

### PR DESCRIPTION
**Pull request by mitkoyanev**

### CODE REVIEW

**Comment by jhalaa** [comment]
Could you delete the console.logs and the commented code?

File: src/components/DisplayComponent.js [hunk 1]
```diff
@@ -113,6 +121,24 @@ class DisplayComponent extends Component {
...
 
+  handleChangeSearch = event => {
+    let search = event.target.value.toLowerCase();
+    search = search.replace('\\', '');
+    search = search.replace('*', '');
+    this.setState(
+      {
+        searchBoxText: search
+      },
+      this.apiCall
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: I dont see this method anywhere, can this be removed?

File: src/components/DisplayComponent.js [hunk 2]
```diff
@@ -254,13 +299,47 @@ class DisplayComponent extends Component {
         <Normalization />
         <ToolkitProvider keyField="id" data={data} columns={columns} exportCSV>
           {props => (
-            <div>
+            <div className="">
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: Empty `className`

File: src/components/DisplayComponent.js [hunk 3]
```diff
@@ -17,6 +18,10 @@ class DisplayComponent extends Component {
       currentPopulation: 0,
       currentLevel: null,
       data: {},
+      curPage: 1,
+      totalPageNumber: 10, //0,
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: Why is total page hardcoded to 10?

File: src/components/DisplayComponent.js (hunk 1 above)
Comment (mitkoyanev) [reply src/components/DisplayComponent.js]: yes.

File: src/components/DisplayComponent.js (hunk 2 above)
Comment (mitkoyanev) [reply src/components/DisplayComponent.js]: here, i want to add something to move the div to the line above. I'm unsure of what, though

File: src/components/DisplayComponent.js (hunk 3 above)
Comment (mitkoyanev) [reply src/components/DisplayComponent.js]: it's a placeholder. It gets overwritten as soon as the code runs.

File: src/components/DisplayComponent.js (hunk 2 above)
Comment (steventhan) [reply src/components/DisplayComponent.js]: I think this is taken care of in #136

File: src/components/DisplayComponent.js (hunk 3 above)
Comment (steventhan) [reply src/components/DisplayComponent.js]: If it's the case then I think we should make it 0 to avoid confusion.

Comment (mitkoyanev) [reply src/components/DisplayComponent.js]: ok.

File: src/components/DisplayComponent.js (hunk 2 above)
Comment (mitkoyanev) [reply src/components/DisplayComponent.js]: Then I'll remove it for now.

**Review by steventhan — APPROVED** [review]

**Pull request merged**